scripts/validate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# usage: ./validate.sh bugid.bug
+# 
+# install: https://github.com/any-json/any-json 
+# install: https://github.com/jessedc/ajv-cli
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SCHEMA="${DIR}/robust.json"
+echo Checking schema compliance for $1
+any-json --input-format=yaml $1 | ajv --verbose --errors=js -s ${SCHEMA} -d /dev/stdin | sed "s/\/dev\/stdin/File/"
```

Comment:
```suggestion
any-json --input-format=yaml $1 | ajv --verbose --errors=text -s ${SCHEMA} -d /dev/stdin | sed "s/\/dev\/stdin/File/"
```

We're humans and we don't have an automated reporting solution in place to parse the JS output for us.

`text` gives us:

> human readable error messages with data paths

Reply:
I thought you would prefer structured messages (because I thought that maybe travis knows what to do with them). If not, I prefer human readable too :)